### DISCUSSION TITLE
Basic staff: Multiple select on a register form does not work

### DISCUSSION THREAD

**Discussion by miguelgargallo** [pocketbase/pocketbase#3557] [Q&A]
Is there anywhere how to make of use of "Multiple" on select options?

`{   "errorDetails": "jobs: Invalid value [\"Modelo de imagen\",\"Azafatas\",\"Cocinero\"]; "errorMessage": "Failed to create record." }`

**Comment by ganigeorgiev** [comment]
I'm not sure that I understand what do you mean.

Could you provide an example how you are sending the request and export your collections configuration as json (Admin UI > Settings > Export collections)?

**Comment by miguelgargallo** [comment]
I have a form, I select multiple options, they are listed from the array, but i don't know how to send this to db and is saying me this error

so, can you provide me some feedback to achive this? I am on JS/TS

**Reply by ganigeorgiev** [reply]
I don't understand what do you mean by _"listed from the array"._

Also how is this form submitted? Is it a regular HTML POST form submit action or you are submitting the form with JS?

Here are couple examples:

1. Regular HTML POST form submit:
    ```html
    <form action="http://127.0.0.1:8090/api/collections/example/records" method="POST" enctype="multipart/form-data">
        <select name="yourSelect" multiple>
            <option value="a">a</option>
            <option value="b">b</option>
            <option value="c">c</option>
            <option value="d">d</option>
        </select>

        <button type="submit">Submit</button>
    </form>
    ```

2. OR submit the form with JavaScript using the JS SDK:
    ```js
    await pb.collection("example").create({
        "yourSelect": ["a", "b"],

        // or you can also use the +/- suffixes to append/remove to existing values in case of an update
        "yourSelect+": ["c", "d"],
    })
    ```
    
If the above doesn't help, please provide a code sample how you are sending the request and export your collections as JSON (_Admin UI > Settings > Export collections_).

**Comment by miguelgargallo** [comment]
I am not being able to make of use of multiple option on select form on a formdata

![image](https://github.com/pocketbase/pocketbase/assets/5947268/5d65b0fe-2b53-4772-a6dc-58bbf2bc7b83)

**Reply by ganigeorgiev** [reply]
I think we are going in circles. Until you provide an example how you are sending the request I don't think I can help you.

**Reply by ganigeorgiev** [reply]
If you use a `FormData` object as body for the request, then multiple values must be submitted by calling `yourData.append("jobs", value)` for each entry.
